cli/bench_command.go

```diff
@@ -127,17 +136,23 @@ func (c *benchCmd) bench(_ *kingpin.ParseContext) error {
 		return fmt.Errorf("number of messages should be greater than 0")
 	}
 	if c.js && c.numSubs > 0 && c.pull {
-		log.Print("JetStream pull consumer mode: subscriber(s) will explicitly acknowledge the consumption of messages")
+		log.Print("JetStream durable pull consumer mode, subscriber(s) will explicitly acknowledge the consumption of messages")
+	}
+	if c.js && c.numSubs > 0 && c.pushDurable {
+		if c.pull {
+			log.Fatal("the durable consumer must be either pull or push, it can not be both")
```

Comment:
should be `The` for consistency, though there's quite a spread of mixing lower/upper case in these log lines, not sure if thats on purpose perhaps?

Reply:
I homogenized so now all error messages start with a capital letter. Some people told me 'in Go error messages should not start with an upper case' but I since checked and that's only for Errorf to cover the case the error message in question is being bubbled up the stack where it is then included in the error log (i.e. `log.Fatalf("This error happened: %v\n",err)` so the error string should not start with a capital). In this case here my error messages are all log.Fatalf directly so they should all be starting with upper case. (source: https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

Reply:
correct, where they are essentially the final result of the error as logged is shown to the user it is fine to be capital, but if say you logged the error value `log.Fatalf("Thing failed: %v", err)` its better error strings are lower case.

Anyway, i dont care if its upper or lower I care more about consistency, consistently "incorrect" throughout is actually better than being a mix of things :)